tools/migration/import

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/python
+import json
+from optparse import OptionParser
+import os
+import urllib2
+import sys
+import logging
+import logging.config
+
+logging.basicConfig(filename="import_project.log", level=logging.INFO)
+logger = logging.getLogger()
+
+class Parameters(object):
+    def __init__(self):
+        self.admiral_endpoint = ''
+        self.admiral_token = ''
+        self.projectsfile = ''
+        self.init_from_input()
+
+    @staticmethod
+    def parse_input():
+        usage = "usage: %prog [options] <admiralendpoint> <token> <projectsfile>"
+        parser = OptionParser(usage)
+        parser.add_option("-a", "--admiralendpoint", dest="admiral_endpoint", help="admiral endpoint")
+        parser.add_option("-t", "--token", dest="admiral_token", help="admiral token")
+        parser.add_option("-f", "--projectsfile", dest="projectsfile", help="the path of exported json file")
+
+        (options, args) = parser.parse_args()
+        return (options.admiral_endpoint, options.admiral_token, options.projectsfile)
+
+    def init_from_input(self):
+        (self.admiral_endpoint, self.admiral_token, self.projectsfile) = Parameters.parse_input()
+
+class Project:
+    def __init__(self, name, public):
+        self.project_name = name
+        self.public = public
+
+class Admiral:
+    def __init__(self, admiral_url, token):
+        self.admiral_url = admiral_url + '/projects'
+        self.token = token
+
+    def __import_project(self, project, retry=True):
+        project_data = json.dumps({ "name": project.project_name, "isPublic": project.public })
```

Comment:
Need default values for project level content trust, auto scan, vulnerability threshold...